pkg/workloads/cortex/client/cortex/client.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2019 Cortex Labs, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pathlib
+import os
+import types
+import subprocess
+import sys
+import shutil
+import yaml
+import urllib.parse
+
+import dill
+import requests
+from requests.exceptions import HTTPError
+
+
+class Client(object):
+    def __init__(
+        self, aws_access_key_id, aws_secret_access_key, operator_url, workspace="./workspace"
+    ):
+        self.operator_url = operator_url
+        self.workspace = workspace
+        self.aws_access_key_id = aws_access_key_id
+        self.aws_secret_access_key = aws_secret_access_key
+        self.headers = {
+            "CortexAPIVersion": "master",
+            "Authorization": "CortexAWS {}|{}".format(
+                self.aws_access_key_id, self.aws_secret_access_key
+            ),
+        }
+
+        pathlib.Path(self.workspace).mkdir(parents=True, exist_ok=True)
+
+    def deploy(self, deployment_name, api_name, model_path, pre_inference, post_inference):
```

Comment:
Should we use the same fields as the names of the fields in yaml, e.g `model` instead of `model_path`?